src/dbus.c

```diff
@@ -328,12 +433,16 @@ static void dbus_cb_dunst_RuleEnable(GDBusConnection *connection,
                 target_rule->enabled = true;
         else if (state == 2)
                 target_rule->enabled = !target_rule->enabled;
+>>>>>>> upstream/master
```

Comment:
There are still version control markers here and in other places

Reply:
oops....

Reply:
anyway fixed